djangocms_transfer/utils.py

```diff
@@ -22,10 +22,6 @@ def get_related_fields(model):
     return [field.name for field in fields if field.is_relation]
 
 
-@lru_cache()
```

Comment:
This is actually the "official" way to get a plugin class. I would keep it instead of using the utils which might change in the future...